Classes/KIFTypist.h

```diff
@@ -13,7 +13,10 @@
 + (void)registerForNotifications;
 + (BOOL)keyboardHidden;
 + (BOOL)enterCharacter:(NSString *)characterString;
+
++ (NSTimeInterval)keystrokeDelay;
```

Comment:
Why don't we just make this a regular `@property` we aren't doing anything in the setter/getter